engines/winnative/payload-schema.yml

```diff
@@ -0,0 +1,57 @@
+id: "http://schemas.taskcluster.net/generic-worker/v1/payload.json#"
+$schema: "http://json-schema.org/draft-04/schema#"
+title: "Payload"
+description: "This schema defines the structure of the `payload` property referred to in a Task Cluster Task definition."
+type: "object"
+required: 
+  - "command"
+  - "maxRunTime"
+properties: 
+  command: 
+    title: "Commands to run"
+    type: "array"
+    minItems: 1
+    items: 
+      type: "array"
+      minItems: 1
+      items: 
+        type: "string"
+    description: "One array per command (each command is an array of arguments). Several arrays for several commands."
```

Comment:
just install bash on windows... it's part of mozbuild anyways...

note: you were never intended to specify the entire task script in the `task.payload`. The `task.payload` property is only for specifying the arguments for whatever task script you have. The task script should be part of the image, or a script downloading the task script from hg/git should be part of the image.

Tasks are not scripts. That will just make people sad when they hit arbitrary limits. So we shouldn't let people do that... If people really want to using `&&` then it pretty obvious that this is not the intended design.